java/pom.xml

```diff
@@ -144,8 +144,11 @@
         <skipNativeCopy>false</skipNativeCopy>
         <cxx11.abi.value>ON</cxx11.abi.value>
         <cxx.flags/>
+        <cmake.export.compile.commands>OFF</cmake.export.compile.commands>
         <CUDA_STATIC_RUNTIME>OFF</CUDA_STATIC_RUNTIME>
         <PER_THREAD_DEFAULT_STREAM>OFF</PER_THREAD_DEFAULT_STREAM>
+        <project.build.directory>target/${CUDA_VERSION}</project.build.directory>
```

Comment:
not exactly sure if we should be modifying this variable. @tgravescs @revans2 @jlowe please comment

Reply:
Just a note, this isn't required for `rapids-compose`. The script in compose only cares about the C++ build dir, and is setting `-Dnative.build.path=` as part of the `mvn package` command.

Reply:
Yes, but this will help manage the java bindings in a similar way cpp build is managed. 
I still need to add the branch name to this property